helpers.go

```diff
@@ -58,6 +59,24 @@ func uuidString(uuid *[C.BTRFS_UUID_SIZE]C.u8) string {
 	return fmt.Sprintf("%x-%x-%x-%x-%x", b[:4], b[4:4+2], b[6:6+2], b[8:8+2], b[10:16])
 }
 
+func le16ToNative(le16 C.__le16) uint16 {
```

Comment:
How about setting the implementation in init() instead of having `if` here? The result of native_endian.NativeEndian wouldn't change throughout the lifecycle of this package.

Reply:
Not sure. I don't see a significant benefit.